index.html

```diff
@@ -420,7 +420,7 @@ <h4>Correctness properties</h4>
 <tr><td><code>a:X</code></td><td><em>X</em> is B</td><td>W</td><td>d=d<sub>X</sub>; u=u<sub>X</sub></td></tr>
 <tr><td><code>s:X</code></td><td><em>X</em> is Bo</td><td>W</td><td>d=d<sub>X</sub>; u=u<sub>X</sub></td></tr>
 <tr><td><code>c:X</code></td><td><em>X</em> is K</td><td>B</td><td>o=o<sub>X</sub>; n=n<sub>X</sub>; d=d<sub>X</sub>; u</td></tr>
-<tr><td><code>d:X</code></td><td><em>X</em> is Vz</td><td>B</td><td>o; n; d</td></tr>
+<tr><td><code>d:X</code></td><td><em>X</em> is Vz</td><td>B</td><td>o; n; d; <small>(Tapscript only)</small> u</td></tr>
```

Comment:
In the gist is says

```
d: remains available, and corresponds to the same script, but has different typing rules in tapscript miniscript (it always has the u type property)
```

Maybe add a comment below `Then there are 5 type modifiers, which guarantee additional properties:`.

For some fragments like `andor(X,Y,Z)` there's a complicated rule of how `d` and `u` are taken from the original. It's non-obvious to me that this guarantees "[d] always has the u type property". Can you clarify that?

Reply:
`andor()` has rules for both the `u` and `d` property. It's unclear how those rules guarantee that any `d` also has `u`.